inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
Hmm, that sounds more like another bug.  Would it work if we seperate that and fix it later? As a fix I'd suggest adding a new function or modify an existing which handles profilefields and their permissions. Not sure though as I haven't worked with them for quite some time.